tests/queries/0_stateless/01560_crash_in_agg_empty_arglist.sql

```diff
@@ -1 +1,5 @@
+-- make sure the system.query_log table is created
+SELECT 1;
+SYSTEM FLUSH LOGS;
```

Comment:
Hmm, why do we need this?

Reply:
If you run single test it may not have system.query_log created yet (it's created automatically when some logging happen). So your test will obviously fail if there is no table.

Reply:
Thanks!

Reply:
FWIW Actually since #11615 `select 1` is not required anymore